lib/runtime/wazero/imports_test.go

```diff
@@ -369,6 +369,41 @@ func Test_ext_crypto_ecdsa_verify_version_2_Table(t *testing.T) {
 	}
 }
 
+func Test_ext_crypto_ecdsa_generate_version_1(t *testing.T) {
+	// TODO: fix this
```

Comment:
Will this be addressed in this PR?

Reply:
Nope,
We need to use a new version of HOST_API_TEST_RUNTIME. We can fix this later.

Reply:
Nope,
We need to add this function in the [host API tester](https://github.com/ChainSafe/polkadot-spec/tree/master/test/runtimes/hostapi) in `chainsafe/polkadot-spec` repo

Reply:
Okay could we create an issue or have some way to track this then?

Reply:
> We need to add this function in the host API tester in chainsafe/polkadot-spec repo

For this you have to create a PR on chainsafe/polkadot-spec repo. Here is a similar PR for reference https://github.com/ChainSafe/polkadot-spec/pull/15

Reply:
We already had PR for this from an outside contributor. I merged it.